### PR TITLE
feat: add articles rail overview

### PR DESCRIPTION
**Pull request by ivan-badyulya**
Created articles rail on overview page

JIRA - https://artsyproduct.atlassian.net/browse/PX-3839

screenshot: 
Screen Shot 2021-04-01 at 5 23 32 PM

### CODE REVIEW

File: src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const ArticlesRail: React.FC<ArticlesRailProps> = ({
+  articles,
+  partnerSlug,
+}) => {
+  return (
+    <>
+      <Flex justifyContent="space-between" alignItems="center">
+        <Text variant="title">Articles</Text>
+        <Sans fontSize={14} size="2" color="black60">
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx:22]: `Sans` is deprecated — only use `Text` and [the v2 variants](https://palette-storybook.artsy.net/?path=/story/components-text--variants)

File: src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <a href={`${sd.APP_URL}/partner2/${partnerSlug}/articles`}>
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx:23]: Use `RouterLink`

Comment (dzucconi) [reply src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx:23]: Also you shouldn't have to make these absolute links with the APP_URL; just a relative link is fine.

**Review by kiryl-zubarau — APPROVED** [review]

File: src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx (hunk 1 above)
Comment (ivan-badyulya) [reply src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx]: fixed

File: src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx (hunk 2 above)
Comment (ivan-badyulya) [reply src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx]: fixed

File: src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+const ArticlesRail: React.FC<ArticlesRailProps> = ({
+  articles,
+  partnerSlug,
+}) => {
+  return (
+    <>
+      <Flex justifyContent="space-between" alignItems="center">
+        <Text variant="title">Articles</Text>
+        <RouterLink to={`/partner2/${partnerSlug}/articles`}>
+          <Text variant="text" color="black">
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/Overview/ArticlesRail.tsx:23]: Minor thing: `black100`

**Review by dzucconi — APPROVED** [review]

**Pull request merged**